src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/FileMetadataFields.scala

```diff
@@ -19,6 +19,7 @@ object FileMetadataFields {
 
   val SHA256ServerSideChecksum = "SHA256ServerSideChecksum"
 
+  case class AddFileMetadataInput(fileId: UUID, value: String, userId: UUID, filePropertyName: String)
```

Comment:
I would place this case class in the `FileService` as its not really used by the GraphQl queries/mutations. Its more a convenience case class for passing data into the repository as opposed to being used by clients calling the API.

Reply:
I've moved the case class to the `FileMetadataService` since it's for passing data to the FileMetadataRepository